project/Common.scala

```diff
@@ -111,8 +111,10 @@ object Common extends AutoPlugin {
       case _ => Seq.empty[String]
     }),
     autoAPIMappings := true,
-    apiURL := Some(url(
-      s"https://pekko.apache.org/api/pekko-connectors/${version.value}/org/apache/pekko/stream/connectors/index.html")),
+    apiURL := {
```

Comment:
Taken from https://github.com/apache/incubator-pekko-http/blob/75f26d6c45d4f0ea78737d8daf96020b3327f703/build.sbt#L37-L40 so we know it should work.